src/demo/OceanObject.js

```diff
@@ -351,8 +372,9 @@ export class FishOceanObject extends OceanObject {
     this.drawFishComponent(this.body, bodyAnchor, ctx);
     this.drawFishComponent(this.scales, bodyAnchor, ctx);
     this.drawFishComponent(this.pectoralFinFront, bodyAnchor, ctx);
-    this.drawFishComponent(this.mouth, bodyAnchor, ctx);
-    this.drawFishComponent(this.eye, bodyAnchor, ctx);
+    //this.drawFishComponent(this.mouth, bodyAnchor, ctx);
```

Comment:
Usual comment: can we get a comment on why this is commented-out for future reference, or just remove it instead?

Reply:
I'll remove these 😬